CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contributing to the AeroGear Android Showcase Template
+
+The AeroGear Android Showcase Template is part of the [AeroGear project](https://aerogear.org/), see the [Community Page](https://aerogear.org/community) for general guidelines for contributing to the project.
+
+This document details specifics for contributions to the Android Showcase Template.
+
+## Issue tracker
+
+The tracking of issues for the AeroGear Android Showcase Template is done in the [AeroGear Project](https://issues.jboss.org/projects/AEROGEAR/issues) in the [JBoss Developer JIRA](https://issues.jboss.org).
+
+See the [AeroGear JIRA Usage and Guidelines Guide](https://aerogear.org/docs/guides/JIRAUsage/) for information on how the issue tracker relates to contributions to this project.
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the Aerogear team is available for technical advice or feedback. 
+You can reach us at [#aerogear](ircs://chat.freenode.net:6697/aerogear) on [Freenode IRC](https://freenode.net/) or the 
+[aerogear google group](https://groups.google.com/forum/#!forum/aerogear)
+-- both are actively monitored.
+
+# Developing the Android Showcase Template
+
+## Prerequisites
+
+Ensure you have the following installed in your machine:
+
+- [Java Development Kit](https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html)
+- [Android Studio and Android SDK](https://developer.android.com/studio/index.html) check [build.gradle](./app/build.gradle) file for the required SDK versions and build tools version
+- [Git SCM](http://git-scm.com/)
+
+## Cloning the repository
+
+```bash
+git clone git@github.com:aerogear/android-showcase-template.git
+cd aerogear-showcase-template/
+```
+
+## Installing dependencies and building the ShowCase Template
+
+To run all the tests:
+```bash
+./gradlew test connectedAndroidTest
+```
+To build:
+```bash
+/gradlew assembleDebug
+```
+To do release build (you will need to setup the signing credential):
+```bash
+./gradlew aR
+```
+See the [Gradle Documentation](https://docs.gradle.org/current/userguide/pt02.html) for more information on Gradle and the Gradle Wrapper
+
+## Using a Published SDK
+
+### jcenter()
+
+To use a published version of the SDK, add jcenter to the [build.gradle](https://github.com/aerogear/android-showcase-template/blob/master/build.gradle) file in the root directory of this project.
+
+```groovy
+allprojects {
+    repositories {
+        jcenter() // <-- Add This line
+        google()
+    }
+}
+```
+
+### Reference the Dependencies
+
+In the [build.gradle](./app/build.gradle) file in the app directory, add the dependencies provided by the SDK.
+
+```groovy
+dependencies {
+    ...
+    implementation 'org.aerogear:core:0.1.0-2018-09'
+    implementation 'org.aerogear:auth:0.1.0-2018-09' // <- adjust to suit release version
```

Comment:
I know there is a comment saying to adjust to suit release version, but I don't like to add any version on the docs, because people usually use it on a copy and past, I always prefer to do some instruction like: 

```groovy
implementation "org.aerogear:auth:[version]"
```

Because people will get an error and will be forced to update this.